mopidy_alsamixer/mixer.py

```diff
@@ -70,13 +74,20 @@ def get_volume(self):
         if not channels:
             return None
         elif channels.count(channels[0]) == len(channels):
-            return int(channels[0])
+            unadjusted_volume = channels[0]
+            if self.logarithmic_volume:
+                unadjusted_volume = 100.0 * math.pow(unadjusted_volume / 100.0, 3.0)
```

Comment:
FYI: https://lazka.github.io/pgi-docs/#GstAudio-1.0/classes/StreamVolume.html#GstAudio.StreamVolume.convert_volume exists which we could use instead of having the conversion functions our selves. For example:

```
GstAudio.StreamVolume.convert_volume(GstAudio.StreamVolumeFormat.LINEAR, GstAudio.StreamVolumeFormat.DB, 70.0)
```

Also if you don't think we should use the Gst provided one, please make the conversion into an internal function instead of duplicating it again on line 89.